Userland/Libraries/LibCore/DateTime.cpp

```diff
@@ -106,7 +106,12 @@ String DateTime::to_string(StringView format) const
     const int format_len = format.length();
 
     auto format_time_zone_offset = [&](bool with_separator) {
+#ifndef __FreeBSD__
         auto offset_seconds = -timezone;
+#else
+        auto raw_time = time(0);
+        auto offset_seconds = timegm(localtime(&raw_time)) - raw_time;
```

Comment:
This looks a bit iffy. Can you expand on why you need this?

Reply:
timezone isn’t working on FreeBSD. timezone is a function in FreeBSD. I got this fix from BertalanD on Discord.

Reply:
Does anything go horridly wrong if we just set `offset_seconds = 0` on FreeBSD? I'd rather err on the side of the simplest solution for Lagom-only workarounds changes like this

Reply:
We could try that, sure.